### PR TITLE
Fixed the creation of temp. folders.

### PR DESCRIPTION
**Pull request by therearesomewhocallmetim**
Added DeprecationWarning type.

### CODE REVIEW

File: tools/python/integration_tests_runner.py [hunk 1]
```diff
@@ -84,12 +84,10 @@ def exec_test(a_tuple):
     test_file, test_name, params = a_tuple
     params[FILTER_KEY] = test_name
 
-    if test_file in TEMPFOLDER_TESTS:
-        out, err, result = exec_test_with_temp(test_file, params)
+    if path.split(test_file)[-1] in TEMPFOLDER_TESTS:
```
Comment (burivuh) [inline tools/python/integration_tests_runner.py]: path.basename?

File: tools/python/integration_tests_runner.py [hunk 2]
```diff
@@ -84,12 +84,10 @@ def exec_test(a_tuple):
...
 
-    if test_file in TEMPFOLDER_TESTS:
-        out, err, result = exec_test_with_temp(test_file, params)
+    if path.basename(test_file) in TEMPFOLDER_TESTS:
+        return exec_test_with_temp(test_file, params)
     else:
-        out, err, result = exec_test_without_temp(test_file, params)
-
-    return test_file, err, result
+        return exec_test_without_temp(test_file, params)
```
Comment (ygorshenin) [inline tools/python/integration_tests_runner.py]: No else after return

**Comment by ygorshenin** [comment]
LGTM

**Comment by burivuh** [comment]
LGTM

**Pull request merged**